src/Compilers/Core/Portable/Serialization/ObjectWriter.cs

```diff
@@ -69,16 +69,18 @@ internal sealed partial class ObjectWriter : IDisposable
         /// Creates a new instance of a <see cref="ObjectWriter"/>.
         /// </summary>
         /// <param name="stream">The stream to write to.</param>
-        /// <param name="cancellationToken"></param>
+        /// <param name="leaveOpen">True to leave the <paramref name="stream"/> open after the <see cref="ObjectWriter"/> is disposed.</param>
+        /// <param name="cancellationToken">Cancellation token.</param>
         public ObjectWriter(
             Stream stream,
+            bool leaveOpen = true,
```

Comment:
Yes, that would be ideal, I'll look into changing the default. I did not do it in this PR since previously ObjectWriter did not dispose the underlying stream at all. I'll need to review all usages to see when the stream should be disposed.